ios/FluentUI/Core/Theme/Tokens/GlobalTokens.swift

```diff
@@ -19,25 +19,82 @@ public final class GlobalTokens {
         case tint20
         case tint30
         case tint40
+        case comm10
+        case comm20
+        case comm30
+        case comm40
+        case comm50
+        case comm60
+        case comm70
+        case comm80
+        case comm90
+        case comm100
+        case comm110
+        case comm120
+        case comm130
+        case comm140
+        case comm150
+        case comm160
     }
+
     lazy public var brandColors: TokenSet<BrandColorsTokens, DynamicColor> = .init { token in
         switch token {
         case .primary:
-            return DynamicColor(light: ColorValue(0x0078D4), dark: ColorValue(0x0086F0))
+            // Communication Blue 80 and Communication Blue 90
+            return DynamicColor(light: ColorValue(0x0F6CBD), dark: ColorValue(0x2886DE))
```

Comment:
nit: I think this might be a bit clearer on one line as `return DynamicColor(light: ColorValue(0x0F6CBD) /* comm80 */, dark: ColorValue(0x2886DE) /* comm90 */)`,

Reply:
Though, that being said, I think it's a bit odd to be defining the color twice. Not sure there's really a better way, and I think .primary - .tint40 are on the way out, so maybe not worth figuring out, either.